### PR TITLE
Audit URL Support

### PR DESCRIPTION
**Pull request by ankit-privado**

### CODE REVIEW

File: src/main/scala/ai/privado/audit/AuditReportConstants.scala [hunk 1]
```diff
@@ -120,4 +122,10 @@ object AuditReportConstants {
 
   val JS_ELEMENTS_TO_BE_EXCLUDED =
     "^(?i)(modules?|loggers?|val|console|require|_|get|key|value|data|page|url|Set|filter|<init>|err|errors|axios|express|router|component|Instance|utils?|app|undefined|context|process|...props?|async|await|const|let|var|this)$"
+
+  val URLREGEX = ".*(://).*"
```
Comment (HiteshMah-Jan) [inline src/main/scala/ai/privado/audit/AuditReportConstants.scala:126]: PRIVADO_URL_SERVER="privado.ai/urls"

Comment (khemrajrathore) [reply src/main/scala/ai/privado/audit/AuditReportConstants.scala:126]: Also include all literals which have `/` in them 
cc - @HiteshMah-Jan

Comment (ankit-privado) [reply src/main/scala/ai/privado/audit/AuditReportConstants.scala:126]: okay, i will include all the literal having `/` in it.

Comment (ankit-privado) [reply src/main/scala/ai/privado/audit/AuditReportConstants.scala]: done @khemrajrathore @HiteshMah-Jan please take a look.

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**